validation/telemetry/heartbeat.4.sample.pass.json

```diff
@@ -3,6 +3,7 @@
   "id": "d059fd44-534e-df49-8860-7d3c93b545e1",
   "creationDate": "2017-07-14T15:19:31.242Z",
   "version": 4,
+  "engagementType": "star",
```

Comment:
Looks like the tests are failing because this should be `stars` - or should the enum include `star`?